src/components/Tube.vue

```diff
@@ -0,0 +1,71 @@
+<!-- TODO: move inline styles to css -->
+<template>
+  <b-list-group-item draggable="true" v-on:dragstart="drag(barcode, $event)" v-bind:class="{active: isActive}" v-on:mouseover="isActive = true" v-on:mouseleave="isActive = false">
+    <b-img left src="tube.svg" height="70"></b-img>
+
+    <div class="info">
+      <div class="barcode">
+        {{ barcode}}
+      </div>
+       <div class="sample_names">
+        {{ sample_names }}
+      </div>
+       <div class="tag_oligos">
+        {{ tag_oligos}}
+      </div>
+    </div>
+ </b-list-group-item>
+</template>
+
+<script>
+
+export default {
+  name: 'Tube',
+  props: {
+    barcode: {
+      type: String,
+      required: true
+    },
+    sample_names: {
+      type: String,
+      required: true
+    },
+    tag_oligos: {
+      type: String,
+      required: true
+    }
+  },
+  data () {
+    return {
+      isActive: false
+    }
+  },
+  // TODO: need to add a a test for drag
```

Comment:
Still todo?

Reply:
Yes. I am envisaging that we will start using drag and drop in Saphyr and ONT so we can revisit it then. It definitely needs a test but unsure of how to do it.